server/services/zigbee2mqtt/model/Xiaomi.js

```diff
@@ -21,7 +21,7 @@ const Xiaomi = {
     WSDCGQ01LM: [features.temperature, features.humidity],
     WSDCGQ11LM: [features.temperature, features.humidity, features.pressure],
     RTCGQ01LM: [features.motion],
-    RTCGQ11LM: [features.motion, features.illuminance, features.temperature, features.voltage],
+    RTCGQ11LM: [features.motion, features.illuminance, features.temperature, features.voltage, features.battery],
```

Comment:
It's autodetected by the service (here https://github.com/GladysAssistant/Gladys/blob/3cb07f3753b277a3ac2331327889d919e28efe5b/server/services/zigbee2mqtt/utils/convertDevice.js#L14).